assets/app/styles/_core.less

```diff
@@ -138,6 +138,22 @@ html, body {
   }
 }
 
+.empty-state-msg {
+  background-color: #ffffff;
+  border-radius: @border-radius-sm;
+  box-shadow: @osc-object-box-shadow;
+  font-weight: 300;
+  margin: auto;
+  max-width: 600px;
+  padding: 30px;
+  text-align: center;
+  p {
+    font-size: @font-size-large;
```

Comment:
@liggitt Looks OK, except the navbar takes a ton of space.

![openshift_management_console](https://cloud.githubusercontent.com/assets/1167259/7543876/1dab05f6-f596-11e4-9ccc-ebdd3e700b63.png)